### ISSUE TITLE
Create a logo

### ISSUE THREAD

**Issue by aashay28** [aashay28/GitBoard#24]

**Comment by Gallantfarhan** [comment]
hey, can I make a logo for the website?

**Comment by aashay28** [comment]
Yes sure, do you have any suggestions on this ?
Please share

**Comment by Gallantfarhan** [comment]
something like this 
![Screenshot 2023-05-13 185742](https://github.com/aashay28/GitBoard/assets/102584554/c1d1d19c-2fa7-41b7-86ff-bdbce3374478)

**Comment by aashay28** [comment]
> something like this 
> ![Screenshot 2023-05-13 185742](https://github.com/aashay28/GitBoard/assets/102584554/c1d1d19c-2fa7-41b7-86ff-bdbce3374478)
> 

Yes sure, but what if we create a landscape logo, just a suggestion

**Comment by Gallantfarhan** [comment]
sorry, I didn't get it, can you clarify, please what do you mean with landscape?

**Comment by aashay28** [comment]
> sorry, I didn't get it, can you clarify, please what do you mean with landscape?

![cluster](https://github.com/aashay28/GitBoard/assets/108337259/6d3f051b-0399-4e5e-a7be-7d9b172c4f75)

- something like this, logo on left and text on right

**Comment by aashay28** [comment]
> something like this ![Screenshot 2023-05-13 185742](https://user-images.githubusercontent.com/102584554/238135951-c1d1d19c-2fa7-41b7-86ff-bdbce3374478.png)

go for it, task is assigned to you @Gallantfarhan

**Comment by Gallantfarhan** [comment]
sorry for they delay actually my exams are near so i was studying these days. i make these logo's hope you like it.
![Blue Bold Creative Studio Logo](https://github.com/aashay28/GitBoard/assets/102584554/8a96dc9b-bd75-481e-892b-4b62cee87ea3)
![Blue Bold Creative Studio Logo (2)](https://github.com/aashay28/GitBoard/assets/102584554/243d98d1-1321-427a-a8d2-523169d2920d)
![Blue Bold Creative Studio Logo (1)](https://github.com/aashay28/GitBoard/assets/102584554/d89fca7b-aee6-4f4e-bee7-1f0df2325776)

**Comment by aashay28** [comment]
@Gallantfarhan The color of the logo does not match the website's branding. Please provide two to three additional designs so that we can choose the best one.

**Comment by Gallantfarhan** [comment]
what are the colors? i'll provide alternatives also.

**Comment by aashay28** [comment]
> what are the colors? i'll provide alternatives also.

![Screenshot 2023-05-25 at 2 51 16 PM](https://github.com/aashay28/GitBoard/assets/108337259/441ed905-d917-4b8a-be97-f47a41d6e181)

As we have a dark background we need to choose the color accordingly.
